middle_end/flambda/to_cmm/un_cps_env.mli

```diff
@@ -49,7 +49,7 @@ val mk :
   Exported_offsets.t ->
   Exported_code.t ->
   Continuation.t -> Continuation.t ->
-  Var_within_closure.Set.t -> t
+  used_closure_vars:Var_within_closure.Set.t Or_unknown.t -> t
```

Comment:
The doc comment should be updated to reflect the fact that the last argument is now named.

Reply:
Fixed